try{
    let a;
    console.log(a.length);
    console.log("Hi therefrom inside");
}
catch(e){
    console.log("Hi there from catch block")
}

console.log("hi there");